### PR TITLE
readme: include links to our social (twitter + discord)

### PR DESCRIPTION
**Pull request by gneyal**
I updated the readme file to include links to our social (twitter + discord).

### CODE REVIEW

**Comment by masseelch** [comment]
image

Looks nice :-)

**Review by a8m — COMMENTED** [review]
Just saw @masseelch’s comment. Looks good, but please, keep the Discord badge and remove the star from “what is Atlas”.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
+         <br />
+         <a href="https://atlasgo.io/getting-started/?utm_source=github&utm_campaign=github_atlas_readme&utm_medium=social" rel="dofollow"><strong>Explore the docs »</strong></a>
+         <br /><br/>
+          <a href="https://discord.com/invite/zZ6sWVg6NT">Discord</a>
+    ·
+          <a href="https://twitter.com/ariga_io/?utm_source=github&utm_campaign=github_atlas_readme&utm_medium=social">Twitter</a>
+     ·
+          <a href="https://github.com/ariga/atlas/issues">Report a Bug</a>
+    ·
+          <a href="https://github.com/ariga/atlas/discussions">Request a Feature</a>
```
Comment (a8m) [inline README.md:24]: Both links should point to a creation of a new issue, but with option for selecting a template (from issue template). See ent as an example please: https://github.com/ent/ent/tree/master/.github/ISSUE_TEMPLATE

Comment (gneyal) [reply README.md:24]: Do you want this in this PR or is it a different one?

Comment (a8m) [reply README.md:24]: Whatever you prefer.

**Pull request closed without merging**